<?xml version='1.0'?>
<launch>
  <!-- The rviz_satellite plugin needs a map frame for some reason --> 
  <!-- node pkg="tf2_ros" type="static_transform_publisher" name="world_to_map" args="0.5 1.6 0 0 0 0 1 /world /map"/ -->

  <!-- Publish transformation between base_link and imu -->
  <!-- node pkg="tf2_ros" type="static_transform_publisher" name="base_link2imu" args="0 0 0 1 0 0 0 /base_link /imu"/ -->

  <!-- Publish transformation between base_link and laser -->
  <node pkg="tf2_ros" type="static_transform_publisher" name="base_link2laser" args="0.45 0 0 0 0 0 1 base_link laser"/>
  
</launch>
